streaming_jupyter_integrations/magics.py

```diff
@@ -230,7 +231,11 @@ def __flink_execute_sql_file_internal(self, statements: Iterable[str]) -> None:
             asyncio.run(task)
 
     @cell_magic
+    @magic_arguments()
+    @argument("--display-row-kind", help="Whether result row kind should be displayed", action="store_true")
     def flink_execute_sql(self, line: str, cell: str) -> None:
+        args = parse_argstring(self.flink_execute_sql, line)
+        self.display_row_kind = args.display_row_kind
```

Comment:
In my opinion, `display_row_kind` should not be set as `self` but passed by arguments

Reply:
done